src/frontend/src/pages/AdminToolsPage/ProjectsConfig/LinkTypeFormModal.tsx

```diff
@@ -82,7 +91,11 @@ const LinkTypeFormModal = ({ showModal, handleClose, defaultValues, onSubmit, li
         <Grid item xs={6}>
           <FormControl fullWidth>
             <FormLabel>LinkType Name</FormLabel>
-            <ReactHookTextField name="name" control={control} />
+            {creatingNew ? (
+              <ReactHookTextField name="name" control={control} />
```

Comment:
you could just have one of these reacthooktextfields, and then set the new disabled prop using: `disabled={defaultValues}`